backend/typescript/graphql/types/taskType.ts

```diff
@@ -0,0 +1,63 @@
+import { gql } from "apollo-server-express";
+
+const taskType = gql`
+  enum Status {
+    PENDING_APPROVAL
+    INCOMPLETE
+    COMPLETE
+    EXCUSED
+  }
+
+  enum Recurrence_Frequency {
+    DAILY
+    WEEKLY
+    BI_WEEKLY
+  }
+
+  input InputTaskDTO {
+    categoryId: Int!
+    title: String
+    status: Status
+    description: String
+    assigneeId: Int!
+    assignerId: Int!
```

Comment:
I think assigner should definitely not be nullable (it's just whoever is making the task). Reconsider/get input regarding assignee (if a staff wants to make a task then assign it later?) 
Same in TaskDTO